torch/export/exported_program.py

```diff
@@ -69,6 +69,7 @@
     OutputKind,
     OutputSpec,
     SymIntArgument,
+    SymFloatArgument,
```

Comment:
Can you check if it's enough? I think it was not only the `import` as I've added another branch in the condition :arrow_down:

Reply:
I think this should be ok?